tests/test_experiment_server.py

```diff
@@ -541,49 +540,40 @@ def test_prevent_duplicate_participant_for_worker(self, a, db_session, webapp):
         assert resp.status_code == 403
 
     def test_notifies_recruiter_when_participant_joins(self, webapp):
-        from dallinger.recruiters import Recruiter
         from dallinger.models import Participant
 
         worker_id = '1'
         hit_id = '1'
         assignment_id = '1'
 
-        with mock.patch('dallinger.experiment_server.experiment_server.recruiters') as recruiters:
-            mock_recruiter = mock.Mock(spec=Recruiter)
-            recruiters.for_experiment.return_value = mock_recruiter
+        target = 'dallinger.recruiters.HotAirRecruiter.notify_recruited'
```

Comment:
In retrospect it seems obvious, but it never occurs to me to do this much simpler patch!